src/main/java/net/atos/client/zgw/zrc/ZRCClientService.java

```diff
@@ -254,6 +255,17 @@ public Zaak updateZaakPartially(final UUID zaakUUID, final Zaak zaak, final Stri
         return zrcClient.zaakPartialUpdate(zaakUUID, zaak);
     }
 
+    /**
+     * Update or delete the field ZaakGeometrie
+     *
+     * @param zaakUUID           UUID of {@link Zaak}.
+     * @param zaakGeometriePatch {@link ZaakGeometriePatch} with a {@link net.atos.client.zgw.zrc.model.Geometry} that needs to be updated or deleted
+     * @return Updated {@link Zaak}
+     */
+    public Zaak updateZaakGeometrie(final UUID zaakUUID, final ZaakGeometriePatch zaakGeometriePatch) {
```

Comment:
Geef hier Geometry mee als de waarde die de geometry mag krijgen (kan ook null zijn) en maak in deze methode een instantie aan van ZaakGeometryPatch. Dan heb je die class ook weer weg geabstraheerd.